### ISSUE TITLE
Rule for redundant curly braces too strict

### ISSUE THREAD

**Issue by JochenFromm** [bbatsov/rubocop#2022]
This line gives the warning `Redundant curly braces around a hash parameter.`

`index({ ssn: 1 }, { unique: true })`

but the curly braces are not redundant here. It is obviously the correct way to define an index for a Mongoid model http://mongoid.org/en/mongoid/docs/indexing.html

If we do not use curly braces it does not work, because the two hashes merge into one:

`index(ssn: 1, unique: true)`

**Comment by jonas054** [comment]
It's only the last argument that's being flagged.
```
 ~$ rubocop -v
0.32.1
 ~$ rubocop --auto-correct /tmp/2022.rb 
Inspecting 1 file
C

Offenses:

/tmp/2022.rb:1:19: C: [Corrected] Redundant curly braces around a hash parameter.
index({ ssn: 1 }, { unique: true })
                  ^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected, 1 offense corrected
 ~$ cat /tmp/2022.rb 
index({ ssn: 1 }, unique: true)
```

**Comment by bbatsov** [comment]
OK, so this was a false alarm.
